uploadReadCounts.m

```diff
@@ -5,8 +5,8 @@
 
```

Comment:
of course, i would like to see this as a `makeTuples` method of `GeneRPMK` but okay for now.